### PR TITLE
User interface Milestone 2

### PR DESCRIPTION
**Pull request by tigamadou**

### CODE REVIEW

File: bin/main [hunk 1]
```diff
@@ -1,3 +1,137 @@
...
+        puts "Game start"
+        
+        while @playing
+            board      
+            puts " #{player.name} Place your symbol on the board"
+            num = gets.chomp.to_i
+            num = num.to_i
+            num = num- 1
+            @boardPlayer[num]=player.symbol
+        end
```
Comment (stratospherique) [inline bin/main]: - Game loop that you have implemented is missing the followong:
* [ ] `[REQUIRED]` game tells which player turn it is
* [ ] `[REQUIRED]` game asks player to select from available moves
* [ ] `[REQUIRED]` game informs player if selected move is invalid
* [ ] `[REQUIRED]` game informs player if selected move is a winning move
* [ ] `[REQUIRED]` game informs player if selected move is a draw move

- **[SUGGESTION]** to accomplish the above, you can use conditional methods when checking for the validity of a move or when checking if a move is a winning/draw move. You are allowed to incorporate these methods in your loop as if they were implemented in a different class related to the logic of the game which you will work on in the next milestone.

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Impressive start @Team :clap: :clap: However, some changes needs to be implemented in your code in order to get approval for this milestone. check :arrow_down: 

- **[REQUIRED]** Check code comments below, it contains notes that could help refactor your code. :nerd_face: 

### That's it :relaxed: When finished, kindly resubmit another code review request :pray: 

Happy coding :v: 
**_Ahmed Mahfoudh_**

File: bin/main (hunk 1 above)
Comment (tigamadou) [reply bin/main]: Thank you @stratospherique

**Review by whiz25 — APPROVED** [review]
## Good job guys :+1: 
You guys have met the requirements for this milestone so I am approving this one. I could not test your game because of a bug. Please fix that as you continue to the next milestone.
bug

### Martin Afani

**Comment by tigamadou** [comment]
@whiz25 Thank you. The bug has already been fixed. :+1:

**Pull request merged**